### DISCUSSION TITLE
Can you monitor 4 120amp CT sensors?

### DISCUSSION THREAD

**Discussion by ray-matos-mpf** [emporia-vue-local/esphome#107] [Q&A]
I have 400amp service with 2 main 200amp panels, would I be able to use 4 120amp ct sensors?

**Comment by flaviut** [comment]
You can't use one Vue with two panels, since it is against the installation instructions & therefore code.

There are only 3 big CT clamp inputs on each Vue. If you'd like to mod your clamps to use a big clamp on the little inputs, you can do that (although modifying them violates the UL listing & is again against code), but I can't support that. There's discussion about stuff like this in https://github.com/emporia-vue-local/esphome/discussions/59
